discord/emoji.py

```diff
@@ -291,7 +291,7 @@ def is_usable(self):
 
         Parameters
         -----------
-        name: :class:`str`
+        name: Optional[:class:`str`]
```

Comment:
```suggestion
        name: :class:`str`
```

As I've mentioned before, `Optional` = takes in `None` as a valid value. If anything, the docstring below it needs fixing.